packages/react-components/src/components/Wizard.tsx

```diff
@@ -0,0 +1,74 @@
+
+ const handlePrevious = () => {
+    const newIndex = activeButtonIndex - 1;
+    newIndex >= 0 && setActiveButtonIndex(newIndex < 0 ? visibleItems.length - 1 : newIndex);
+ };
+
+ const handleNext = () => {
+    const newIndex = activeButtonIndex + 1;
+    newIndex <= visibleItems.length - 1 && setActiveButtonIndex(newIndex >= visibleItems.length ? 0 : newIndex);
+ };
+
+ const handleClick = () => {
+   setShortDescription(!shortDescription);
+   setLongtDescription(!longDescription);
+};
+
+ const Comp = visibleItems.length > 0 ? getRenderer(visibleItems[activeButtonIndex], mappings) : '';
+
+ return visible ? (
+    <div id={id} className="cmp-adaptiveform-wizard" data-panelcontainer="wizard" data-placeholder-text="Please drag Wizard components here" data-cmp-visible={visible} data-cmp-enabled={enabled}>
+         {label?.visible && <label className="cmp-adaptiveform-wizard__label" htmlFor={id}>{label?.value}</label>}
+         {(props?.description || props?.tooltip) && <button className="cmp-adaptiveform-wizard__questionmark" onClick={handleClick}></button>} 
+         {shortDescription && props?.tooltip && <div data-cmp-visible={shortDescription} className='cmp-adaptiveform-wizard__shortdescription'>{props?.tooltip}</div>}
+          <div aria-live="polite">
+             {longDescription && props?.description  && <div data-cmp-visible={longDescription} className="cmp-adaptiveform-wizard__longdescription">{props?.description}</div>}
+         </div>
+           <div className="cmp-adaptiveform-wizard__widget cmp-adaptiveform-wizard__widget--empty">
+            <div className="cmp-adaptiveform-wizard__tabs-container">
+                <ol id={`${id}_guide-item-nav-container`} className="cmp-adaptiveform-wizard__tabList" role="tablist">
+                   {visibleItems.map((item: State<ContainerJson>, index: number) =>( 
+                   <li title={item?.label?.value} id={`${item.id}_wizard-item-nav`} className={index===activeButtonIndex?'cmp-adaptiveform-wizard__tab cmp-adaptiveform-wizard__tab--active': 'cmp-adaptiveform-wizard__tab' } role="tab" tabIndex={index===activeButtonIndex?0:-1} aria-selected={index===activeButtonIndex?'true':'false'} data-cmp-hook-adaptiveformwizard="tab">{localizeTabLabel(item)}</li>
+                    ))}
+               </ol>
+           </div>
+           {Comp && <div className="cmp-adaptiveform-wizard__previousNav" role="navigation" aria-label="Previous Button" onClick={handlePrevious} tabIndex={0}></div>}
+           {Comp && <div data-cmp-hook-adaptiveformtabs="tabpanel" className={activeButtonIndex === visibleItems[activeButtonIndex].index?'cmp-adaptiveform-wizard__wizardpanel cmp-adaptiveform-wizard__wizardpanel--active':'cmp-adaptiveform-wizard__wizardpanel'} role="tabpanel" tabIndex={0} aria-hidden={activeButtonIndex===visibleItems[activeButtonIndex].index?'false':'true'} aria-labelledby={`${visibleItems[activeButtonIndex].id}_wizard-item-nav`} data-cmp-hook-adaptiveformwizard="wizardpanel">
```

Comment:
why is this "tabpanel"??, It won't work with tabpanel, refer wizard.html? 
How is functionality running with this, have we run basic navigation test cases for wizard with this